apps/examples/testcase/ta_tc/eventloop/utc/utc_eventloop_main.c

```diff
@@ -0,0 +1,420 @@
+	(void)eventloop_loop_run();
+	TC_ASSERT_EQ("eventloop_add_timer", el_timer_flag, 1);
+
+	/* Add timer, it is stopped by returning EVENTLOOP_CALLBACK_STOP in callback function after 3 times. */
+	el_timer_flag = 0;
+	repeat_timer = eventloop_add_timer(1000, true, repeat_timer_3times_cb, EL_REPEAT_TIMER_DATA);
+	(void)eventloop_loop_run();
+	TC_ASSERT_EQ("eventloop_add_timer", el_timer_flag, 3);
+
+	/* Add timer which repeats forever, it is stopped by calling eventloop_delete_timer. */
+	el_timer_flag = 0;
+	g_repeat_timer = eventloop_add_timer(1000, true, repeat_timer_cb, EL_REPEAT_TIMER_DATA);
+	/* Add timer which will stop above repeated timer, g_repeat_timer after 5s. */
+	timer = eventloop_add_timer(5000, false, (timeout_callback)repeat_timer_stop_cb, EL_REPEAT_TIMER_DATA);
+	(void)eventloop_loop_run();
+	TC_ASSERT_EQ("eventloop_add_timer", el_timer_flag, 4);
+
+	TC_SUCCESS_RESULT();
+}
+
+static void utc_eventloop_delete_timer_n(void)
+{
+	int ret;
+
+	ret = eventloop_delete_timer(NULL);
+	TC_ASSERT_EQ("eventloop_delete_timer", ret, EVENTLOOP_INVALID_PARAM);
+
+	ret = eventloop_delete_timer(g_repeat_timer);
+	TC_ASSERT_EQ("eventloop_delete_timer", ret, EVENTLOOP_INVALID_HANDLE);
+
+	TC_SUCCESS_RESULT();
+}
+
+static void utc_eventloop_delete_timer_p(void)
+{
+	el_timer_t *timer;
+	int ret;
+	
+	el_timer_flag = 0;
+	timer = eventloop_add_timer(0, false, (timeout_callback)timer_cb, NULL);
+	ret = eventloop_delete_timer(timer);
+	TC_ASSERT_EQ("eventloop_delete_timer", ret, OK);
+
+	/* Try to delete already deleted timer, it will be return EVENTLOOP_INVALID_HANDLE. */
+	ret = eventloop_delete_timer(timer);
+	TC_ASSERT_EQ("eventloop_delete_timer", ret, EVENTLOOP_INVALID_HANDLE);
+
+	TC_SUCCESS_RESULT();
+}
+
+static void utc_eventloop_add_timer_async_n(void)
+{
+	el_timer_t *timer;
+
+	timer = eventloop_add_timer_async(1000, false, NULL, NULL);
+	TC_ASSERT_EQ("eventloop_add_timer_async", timer, NULL);
+
+	TC_SUCCESS_RESULT();
+}
+
+static void utc_eventloop_add_timer_async_p(void)
+{
+	int sleep_cnt;
+	el_timer_t *timer;
+	el_timer_t *repeat_timer;
+
+	el_timer_flag = 0;
+	timer = eventloop_add_timer_async(0, 0, (timeout_callback)timer_cb, EL_TIMER_DATA);
+	sleep_cnt = 0;
+	while (sleep_cnt < 5) {
+		sleep(1);
+		sleep_cnt++;
+		if (el_timer_flag == 1) {
+			break;
+		}
+	}
+	TC_ASSERT_EQ("eventloop_add_timer_async", el_timer_flag, 1);
+
+	el_timer_flag = 0;
+	repeat_timer = eventloop_add_timer_async(1000, true, repeat_timer_3times_cb, EL_REPEAT_TIMER_DATA);
+	sleep_cnt = 0;
+	while (sleep_cnt < 5) {
+		sleep(1);
+		sleep_cnt++;
+		if (el_timer_flag == 3) {
+			break;
+		}
+	}
+	TC_ASSERT_EQ("eventloop_add_timer_async", el_timer_flag, 3);
+
+	TC_SUCCESS_RESULT();
+}
+
+static int wifi_on_callback(void *data, void *event_data)
+{
+	static int cnt = 0;
+
+	printf("WIFI ON CALLBACK!! Callback data: %s\n", (char *)data);
+	if (event_data != NULL) {
+		printf("Event data : %s\n", (char *)event_data);
```

Comment:
Please check wheter this event_data is same with data which sent from eventloop_send_event